bokehjs/src/coffee/common/tool_manager.coffee

```diff
@@ -99,7 +99,7 @@ class ToolManager extends HasProperties
           continue
 
         gestures[et].tools.push(tool)
-        @listenTo(tool, 'change:active', _.bind(@_active_change, tool))
+        @listenTo(tool, 'change:active', () => @_active_change(tool))
```

Comment:
That only partially fixes things, though. The plot at least renderers, but the tool manager and event dispatching are completely broken.